src/entryPoints/HyperSwitchApp.res

```diff
@@ -81,6 +81,13 @@ let make = () => {
     None
   }, [userGroupACL])
 
+  let ompDropdowns =
```

Comment:
see whether you were able to move this under OMP component

Reply:
its causing dependency cycle, as the helper file it itself being used in switch components file. I think since we are using the base components itself , using it here like this should not be an issue